lib/features/steps/document_server_steps.rb

```diff
@@ -0,0 +1,7 @@
+# @!group Document steps
+
+# Starts the document server manually.  It will be stopped automatically at the end of each scenario
+# (if started in this way).
+When('I start the document server') do
```

Comment:
I get why this step exists - but I'm wondering if there's a neater way to accomplish the same things.  I guess at least this is explicit and can be put in a background section?

Reply:
The document server can also be started on the command line or by setting a config value in `env.rb`.  If lots of scenarios needed it, I'd go that route rather than use a Background.  I only did it this way because it's the only scenario that needs the document server and other ways of starting it will keep it running for the whole test suite - but I'm not not precious about it either way.